### ISSUE TITLE
.Magnitude , typecasting of objects (pint example) , and raw data access

### ISSUE THREAD

**Issue by Enkrona** [brendan-w/python-OBD#57]
I have an unmodified library directly cloned from the git and followed directions to a T. Issue is, when using code examples, the .value.to("mph") will not function. Constantly says that: 

```
Traceback (most recent call last):
  <module>
    in_RPM.value.magnitude AttributeError: 'NoneType' object has no attribute 'magnitude'
```

Additionally for the .magnitude referenced in the docs there is this error:

```
 Traceback (most recent call last):
  <module>
    in_RPM.value.to("mph") AttributeError: 'NoneType' object has no attribute 'to'
```

So a this moment, there is no way to get the "raw" data that is just the object's  data value, and able to manipulate the object's data as it is a protected instance variable. I have traced through the code, and have been unable to find a sane answer as to why the object's data is not able to be manipulated. 

Basically, issue is access to raw data as a int, float, str, etc. I am more than happy to contribute any of my commits I change in the process of fixes. 

Thanks.

**Comment by brendan-w** [comment]
Hello!

This is happening because python-OBD is returning null responses. When this happens, `response.value` is set to `None` (instead of a pint quantity), and `response.is_null()` will return True.

Getting null response usually means one of two things:
- python-OBD is protecting you from sending an unsupported command, and returns a null response for consistencies sake.
- You have a bad connection to the car (in which case, all commands will return null).

It is most likely that later. If it's not giving you any obvious errors, then I'd ask you to enable debug mode by adding the following line just after importing the library.

```python
import obd
obd.logger.setLevel(obd.logging.DEBUG) # enables all debug information
# ... re-run the failing code
```

Let me know what that outputs.
